mlflow/tracing/fluent.py

```diff
@@ -576,6 +576,51 @@ def f():
         return None
 
 
+@experimental
+def update_current_trace(
+    tags: Optional[dict[str, str]] = None,
```

Comment:
what would happen if I pass non string values?

Reply:
Good call. It might block the trace to be exported. If that's the case, we should add validation here to gracefully handle invalid tags instead.

Reply:
Well actually we stringfy values in the [set_trace_tag](https://github.com/mlflow/mlflow/blob/2ee3db71712d716bdc3dc06ae301dacf4f652712/mlflow/tracking/_tracking_service/client.py#L398) API.